pkg/reading/status.go

```diff
@@ -0,0 +1,10 @@
+package reading
+
+type Status struct {
+	Id          string `json:"id"`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [struct field Id should be ID](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)